### ISSUE TITLE
build / version info

### ISSUE THREAD

**Issue by Axiom21** [MicrosoftDocs/cpp-docs#4402]
Hi, it would be handy if you provided the build / version info for Visual C++ Redistributable packages, it's not until you've downloaded it, that you can check.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 75e6432b-c6ec-e208-5f1c-bbc4fde9929b
* Version Independent ID: 9cbb940e-cddd-f82f-e8bb-ee62b17af794
* Content: [Latest supported Visual C++ Redistributable downloads](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
* Content Source: [docs/windows/latest-supported-vc-redist.md](https://github.com/MicrosoftDocs/cpp-docs/blob/main/docs/windows/latest-supported-vc-redist.md)
* Product: **visual-cpp**
* Technology: **cpp-windows**
* GitHub Login: @MahmoudGSaleh
* Microsoft Alias: **msaleh**

**Comment by corob-msft** [comment]
@Axiom21 

Thanks for the request. This isn't the first such request, so I'm resolving it as a duplicate of MicrosoftDocs/cpp-docs/issues/4172, which is still open. The docs team isn't notified when the download versions of Redistributable files are updated, so this function has to be taken on by the development team. 

@MahmoudGSaleh for awareness.
